packages/kbn-es/src/cluster.js

```diff
@@ -268,22 +268,25 @@ exports.Cluster = class Cluster {
 
     this._log.debug('%s %s', ES_BIN, args.join(' '));
 
-    options.esEnvVars = options.esEnvVars || {};
```

Comment:
`esEnvVars` was not being passed or referenced anywhere within the repository. Opted to instead use  `ES_JAVA_OPTS`